openapi/v2/support-scheduler.yaml

```diff
@@ -137,32 +167,71 @@ components:
         created:
           description: "A timestamp indicating when the interval action was created."
           type: integer
+        modified:
+          description: "A timestamp indicating when the interval action was last modified."
+          type: integer
+        id:
+          description: "Uniquely identifies the interval action"
+          type: string
+          format: uuid
+        name:
+          description: "Non-database identifier for an interval action"
+          type: string
+        intervalName:
+          description: "The name of the interval to which the action is associated."
+          type: string
         host:
           description: "The host targeted by the action"
           type: string
         httpMethod:
           description: "Indicates which Http verb should be used when the action targets a REST endpoint."
           type: string
+        parameters:
+          description: "Any parameters required by the action"
+          type: string
+        path:
+          description: "The required path at the targeted host for fulfillment of the action."
+          type: string
+        port:
+          description: "The port to address on the targeted host"
+          type: integer
+        protocol:
+          description: "Identifies the protocol required by the action"
+          type: string
+        publisher:
+          type: string
+        target:
+          description: "The target of the action"
+          type: string
+      required:
+        - name
```

Comment:
Shouldn't `Host` and `httpMethod` also be required? We default  `Protocol` to `http` if not set, but without `Host`  and `httpMethod` the REST call will fail.

Reply:
Modified to use the common Address.